specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -88,6 +88,96 @@
         ]
       },
       "x-ms-parameter-location": "method"
+    },
+    "WeatherYear": {
```

Comment:
WeatherYear is always required so it's ok we keep this and add `required: true`

Reply:
Updated